p2p/protocols/accounting.go

```diff
@@ -129,6 +132,7 @@ func (ah *Accounting) Send(peer *Peer, size uint32, msg interface{}) error {
 	}
 	// evaluate the price for sending messages
 	costToLocalNode := price.For(Sender, size)
+	log.Debug(fmt.Sprintf("Sending msg type %v, cost %v", reflect.TypeOf(msg), costToLocalNode))
```

Comment:
For all `log` messages we usually use the format `log.Debug(msg, key1, value1, key2, value2)`.

Thus this would become something like `log.Debug("Sending message", "msg type", reflect.TypeOf(msg), "cost", costToLocalNode)`

Reply:
Fixed.